lusidtools/cocoon/utilities.py

```diff
@@ -837,6 +837,10 @@ def handle_nested_default_and_column_mapping(
         The updated mapping
     """
 
+    # copy the data frame to ensure that it is a copy and not a view (which could make changes to the original
```

Comment:
Could you just add to this comment that the reason for its addition is to remove the `SettingWithCopyWarning` because whilst having a copy is cleaner and better from a coding stand point, this is the real reason it has been added. Also capitalise the first letter of the sentence.